buildbot/osuosl/master/config/builders.py

```diff
@@ -3535,7 +3535,25 @@
                     checkout_llvm_sources=False,
                     script_interpreter=None,
                     clean=True)},
-
+    
+    # Builders that test the premerge configuration
```

Comment:
*I* know that these builders are for post-submit testing of the premerge configuration, but nothing in the names or comments here makes that at all clear.  Ideally something in the builder names themselves would make this clear. Otherwise, at the very least there should be a very good set of comments here making that clear.  We don't want people confusing this set up with the actual premerge testing infrastructure.